firefox-ios/Client/Frontend/PasswordGenerator/PasswordGeneratorMiddleware.swift

```diff
@@ -21,14 +29,25 @@ final class PasswordGeneratorMiddleware {
     }
 
     private func showPasswordGenerator(tab: Tab, windowUUID: WindowUUID) {
-       generateNewPassword(with: tab, completion: { generatedPassword in
-           let newAction = PasswordGeneratorAction(
-               windowUUID: windowUUID,
-               actionType: PasswordGeneratorActionType.updateGeneratedPassword,
-               password: generatedPassword
-           )
-           store.dispatch(newAction)
-       })
+        guard let origin = tab.url?.origin else {return}
```

Comment:
Not super urgent right now, but eventually we will enable password manager/generator for iframes. In that case, mapping a password to `tab.url?.origin` will only work for the top frame and we will be re-using passwords cross-frame for other third-party frames. Can you add a ticket comment here with a jira ticket please. I can mark it as a blocking for the iframe work later. I don't think this is a blocker for this PR though 😄